port/scorecard/refreshScorecardState.go

```diff
@@ -42,5 +86,7 @@ func refreshScorecardState(ctx context.Context, state *ScorecardModel, s *cli.Sc
 	}
 
 	state.Rules = stateRules
-
+	if len(includeLevels) > 0 && includeLevels[0] || (shouldRefreshLevels(state.Levels, s.Levels)) {
```

Comment:
includeLevels[0]  is redundant?

Reply:
when the developer should use the `shouldRefreshLevels` and when the `len(includeLevels)`?